graphql/schema/types/dvd.graphql

```diff
@@ -0,0 +1,51 @@
+type Dvd {
```

Comment:
Convention in go and elsewhere is for classnames that are acronyms should be all in capital - ie this should `DVD`. As such the references to this should changed as well - `allDVD` for example. Where it is in lowercase, it can stay as such - `dvdCreate` is fine for example. This should be changed in the graphql schema and in the golang code.

Reply:
I'm going to try  to adapated to the convention,